app/code/Magento/Security/Model/AdminSessionsManager.php

```diff
@@ -72,20 +77,22 @@ class AdminSessionsManager
      *
      * Means that after session was prolonged
      * all other prolongs will be ignored within this period
+     *
+     * @var int
      */
     private $maxIntervalBetweenConsecutiveProlongs = 60;
 
     /**
      * @param ConfigInterface $securityConfig
-     * @param \Magento\Backend\Model\Auth\Session $authSession
+     * @param Session $authSession
```

Comment:
Can you please add a full class reference here as we are following the same in Core and it's easy to identify class.